src/components/ProseA.vue

```diff
@@ -28,7 +28,7 @@ const props = defineProps({
   },
 });
 
-const isApiReferenceLink = props.href.startsWith('https://docs.centrapay.com/api');
+const isApiReferenceLink = props.href.startsWith('/api');
```

Comment:
Do we still want this API reference badge?

Reply:
I think it might be useful for the guides as the guides were written for people that aren't engineers and they might want to know if they are going to the api. In the api docs we probably don't need it. I'm not sure if there is a way to have it in the guides but not the api docs?

Reply:
Yeah for sure - I'll leave this for now 🙌